libminifi/include/utils/OptionalUtils.h

```diff
@@ -21,17 +21,20 @@
 #include <utility>
 
 #include <nonstd/optional.hpp>
-#include "utils/GeneralUtils.h"
+#include "utils/gsl.h"
 
 namespace org {
 namespace apache {
 namespace nifi {
 namespace minifi {
 namespace utils {
+template <typename T>
+using optional = nonstd::optional<T>;
+const nonstd::nullopt_t nullopt = nonstd::nullopt;
```

Comment:
As pointed out in the other comment, these could become using-declarations.
```suggestion
using nonstd::optional;
using nonstd::nullopt;
```

Reply:
Already modified.